docs/examples/plot_diffuse_fraction.py

```diff
@@ -0,0 +1,186 @@
+dhi = [
+    greensboro.DHI, out_disc.dhi_disc, out_dirint.dhi_dirint,
+    out_erbs.dhi_erbs, out_boland.dhi_boland]
+dhi = pd.concat(dhi, axis=1).rename(columns=dhi_renames)
+ghi_kt = pd.concat([greensboro.GHI/1000.0, out_erbs.kt], axis=1)
+
+# %%
+# Finally, let's plot them for a few winter days and compare
+
+JAN6AM, JAN6PM = '1990-01-04 00:00:00-05:00', '1990-01-07 23:59:59-05:00'
+f, ax = plt.subplots(3, 1, figsize=(8, 10), sharex=True)
+dni[JAN6AM:JAN6PM].plot(ax=ax[0])
+ax[0].grid(which="both")
+ax[0].set_ylabel('DNI $[W/m^2]$')
+ax[0].set_title('Comparison of Diffuse Fraction Estimation Methods')
+dhi[JAN6AM:JAN6PM].plot(ax=ax[1])
+ax[1].grid(which="both")
+ax[1].set_ylabel('DHI $[W/m^2]$')
+ghi_kt[JAN6AM:JAN6PM].plot(ax=ax[2])
+ax[2].grid(which='both')
+ax[2].set_ylabel(r'$\frac{GHI}{E0}, k_t$')
+f.tight_layout()
+
+# %%
+# And a few spring days ...
+
+APR6AM, APR6PM = '1990-04-04 00:00:00-05:00', '1990-04-07 23:59:59-05:00'
+f, ax = plt.subplots(3, 1, figsize=(8, 10), sharex=True)
+dni[APR6AM:APR6PM].plot(ax=ax[0])
+ax[0].grid(which="both")
+ax[0].set_ylabel('DNI $[W/m^2]$')
+ax[0].set_title('Comparison of Diffuse Fraction Estimation Methods')
+dhi[APR6AM:APR6PM].plot(ax=ax[1])
+ax[1].grid(which="both")
+ax[1].set_ylabel('DHI $[W/m^2]$')
+ghi_kt[APR6AM:APR6PM].plot(ax=ax[2])
+ax[2].grid(which='both')
+ax[2].set_ylabel(r'$\frac{GHI}{E0}, k_t$')
+f.tight_layout()
+
+# %%
+# And few summer days to finish off the seasons.
+
+JUL6AM, JUL6PM = '1990-07-04 00:00:00-05:00', '1990-07-07 23:59:59-05:00'
+f, ax = plt.subplots(3, 1, figsize=(8, 10), sharex=True)
+dni[JUL6AM:JUL6PM].plot(ax=ax[0])
+ax[0].grid(which="both")
+ax[0].set_ylabel('DNI $[W/m^2]$')
+ax[0].set_title('Comparison of Diffuse Fraction Estimation Methods')
+dhi[JUL6AM:JUL6PM].plot(ax=ax[1])
+ax[1].grid(which="both")
+ax[1].set_ylabel('DHI $[W/m^2]$')
+ghi_kt[JUL6AM:JUL6PM].plot(ax=ax[2])
+ax[2].grid(which='both')
+ax[2].set_ylabel(r'$\frac{GHI}{E0}, k_t$')
+f.tight_layout()
+
+# %%
+# Conclusion
+# ----------
+# The Erbs and Boland are correlations with only kt, which is derived from the
+# horizontal component of the extra-terrestrial irradiance. Therefore at low
+# sun elevation (zenith ~ 90-deg), especially near sunset, this causes kt to
+# explode as the denominator approaches zero. This is controlled in pvlib by
+# setting ``min_cos_zenith`` and ``max_clearness_index`` which each have
+# reasonable defaults, but there are still concerning spikes at sunset for Jan.
+# 5th & 7th, April 4th, 5th, & 7th, and July 6th & 7th. The DISC & DIRINT
+# methods differ from Erbs and Boland be including airmass, which seems to
+# reduce DNI spikes over 1000[W/m^2], but still have errors at other times.
+#
+# Another difference is that DISC & DIRINT return DNI whereas Erbs & Boland
+# calculate the diffuse fraction which is then used to derive DNI from GHI and
+# the solar zenith, which exacerbates errors at low sun elevation due to the
+# relation: DNI = GHI*(1 - DF)/cos(zenith).
```

Comment:
Suggested revision
```
# horizontal component of the extra-terrestrial irradiance. At low
# sun elevation (zenith near 90 degrees), especially near sunset, kt can
# explode because the denominator (extra-terrestrial irradiance) approaches
# zero. In pvlib this behavior is moderated by ``min_cos_zenith`` and ``max_clearness_index``
# which each have reasonable defaults. Even so, near sunset there are still spikes in DNI
# for Jan. 5th & 7th, April 4th, 5th, & 7th, and July 6th & 7th. 

# By contrast, the DISC and DIRINT methods estimate DNI first by means of
# correlations, which include additional variables such as airmass. These methods
# seem to reduce DNI spikes over 1000 [W/m^2].
```

This statement needs to go, "error" isn't synonymous with different from TMY3 DNI:
```
, but still have errors at other times.
```


I think these points are covered in the edited paragraph above:

```
#
# Another difference is that DISC & DIRINT return DNI whereas Erbs & Boland
# calculate the diffuse fraction which is then used to derive DNI from GHI and
# the solar zenith, which exacerbates errors at low sun elevation due to the
# relation: DNI = GHI*(1 - DF)/cos(zenith).
```